v2.1/configure-zone.md

```diff
@@ -0,0 +1,111 @@
+---
+title: CONFIGURE ZONE
+summary: Use the CONFIGURE ZONE statement to add, modify, reset, and remove replication zones.
+toc: true
+---
+
```

Comment:
General comment: this page does not appear to link to the `SHOW ZONE CONFIGURATION` statement page anywhere.  It might be that the includes that use that statement should link to the page.  I also think the **See Also** section should link to it.  (Unless I'm somehow missing it? In which case apologies).